src/template.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Micro-Tasks</title>
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="assets/icons/favicon/favicon-32x32.png"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="16x16"
+      href="assets/icons/favicon/favicon-16x16.png"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Nunito:400,700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="https://fonts.cdnfonts.com/css/cocogoose" />
+  </head>
+  <body>
+    <main class="flex justify-center w-screen ms:flex-col">
+      <section
```

Comment:
The list of task items is the most important aspect of the application. The current interface only displays a maximum of 3 tasks on the screen without scrolling.

Consider making the app logo branding smaller, and stretch the task container towards the bottom window view to provide room to display more tasks.

Reply:
I will see what I can do about that, great feedback @PraisesPJMT  😄